docs/topics/api/scanners.rst

```diff
@@ -28,3 +28,4 @@ This endpoint returns a list of labelled scanner results.
     :>json string label: Either ``good`` or ``bad``.
     :>json object results: The scanner (raw) results.
     :>json string created: The date the result was created, formatted with `this format <http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.15>`_.
+    :>json string|null model_version: The model version when applicable, ``null`` otherwise.
```

Comment:
this sentence doesn't really mean anything to someone who doesn't understand the whole scanner ecosystem.  What's a model, what does it's version mean, and when is it applicable?

Reply:
Vagueness is a feature here.